src/client/cli/cmd/version.cpp

```diff
@@ -32,16 +33,39 @@ mp::ReturnCode cmd::Version::run(mp::ArgParser* parser)
         return parser->returnCodeFrom(ret);
     }
 
-    cout << "multipass  " << multipass::version_string << "\n";
+    bool format_is_set = parser->isSet("format");
+
+    if (!format_is_set)
+    {
+        cout << "multipass  " << multipass::version_string << "\n";
```

Comment:
This would require updating the `VersionReply` virtual void function in `formatter.h`, and that would then trickle down into all the other `VersionReply` formatters. The issue is we need to toggle the legacy output in the `table` formatter, and this will require passing in a boolean flag to that function to trigger that specific path.

Reply:
Why? Except for the update info, the output of the table formatter is almost identical to the "legacy":

```plain
$ multipass version --format table
multipass:      1.8.0-dev.216.pr2077+ge0d7a7f2
multipassd:     1.8.0-dev.216.pr2077+ge0d7a7f2
$ multipass version               
multipass  1.8.0-dev.216.pr2077+ge0d7a7f2
multipassd 1.8.0-dev.216.pr2077+ge0d7a7f2
```

So let's just make the output of the table formatter equal to the legacy one and we can then rely on it?

Reply:
Got, I misunderstood what was required. I have updated the table format to drop the colon. I have cleaned up the `run` code as well.